src/app/lib/tables.scss

```diff
@@ -0,0 +1,144 @@
+@import "./colors.scss";
```

Comment:
I would recommend pulling some of the customizable parts into variables with !default so they can be customized if so desired https://robots.thoughtbot.com/sass-default.

Reply:
also aliasing the font-awesome characters with variables would make it more readable

Reply:
Great suggestions, I'll work on that